Documentation CYF 1.0/pages/variables.html

```diff
@@ -239,6 +239,8 @@ <h4>Monster script variables</h4>
                 This overrides the random dialogue and is meant for special actions (e.g. you hit Vegetoid's green carrots after selecting Dinner from the 
                 ACT menu). This variable gets <i>cleared every time after it's read out in the monster dialogue phase</i>. This is done
                 so you don't have to take care of managing it manually.</p>
+                <p><span class="CYF"></span> <span class="string"></span> <span class="term">defensemisstext</span> - The text who will be displayed when the <span class="term">def</span> is too high. "MISS" by default</p>
```

Comment:
The text which will be displayed if the Player's attack is successful but deals 0 damage. "MISS" by default.